clang/include/clang/Basic/FPOptions.def

```diff
@@ -26,4 +26,5 @@ OPTION(AllowReciprocal, bool, 1, NoSignedZero)
 OPTION(AllowApproxFunc, bool, 1, AllowReciprocal)
 OPTION(FPEvalMethod, LangOptions::FPEvalMethodKind, 2, AllowApproxFunc)
 OPTION(Float16ExcessPrecision, LangOptions::ExcessPrecisionKind, 2, FPEvalMethod)
+OPTION(FPAccuracy, LangOptions::FPAccuracyKind, 2, Float16ExcessPrecision)
```

Comment:
This requires 3 bits to store all the values of `FPAccuracyKind` (are we missing test coverage? I would have expected that to cause a failure somewhere.)

Reply:
I changed it to be 3 bits, but not sure what testing I should be doing to catch the error? The LIT test in the Driver folder is exercising all values of FPAccuracyKind and was passing with the 2 bits width.